sipa/utils/__init__.py

```diff
@@ -159,6 +159,25 @@ def meetingcal():
     return next_meetings
 
 
+def subscribeStatusPage(url: str, email: str) -> bool | None:
+    try:
+        response = requests.post(url, timeout=1,
+                                 headers={
+                                     "Authorization": "Token " + current_app.config['STATUS_PAGE_AUTH'],
+                                     "Content-Type": "application/json; charset=utf8"
+                                 },
+                                 json={"email": email})
```

Comment:
Ideally, use `darker` or something else to properly format this. Not breaking after an opening parenthesis causes alignment quite far to the right, so just line-breaking and indenting once is the better choice here.
```suggestion
        response = requests.post(
            url,
            timeout=1,
            headers={
                "Authorization": "Token " + current_app.config['STATUS_PAGE_AUTH'],
                "Content-Type": "application/json; charset=utf8"
            },
            json={"email": email}
        )
```
Also, check whether the content-type is necessary; I'd guess that if you pass `json` instead of `data` this should be added automatically.